pkg/test/framework/components/echo/instances.go

```diff
@@ -25,6 +25,19 @@ import (
 // Instances contains the instances created by the builder with methods for filtering
 type Instances []Instance
 
+// Clusters returns a list of cluster names that the instances are deployed in
+func (i Instances) Clusters() []string {
```

Comment:
nit: Should it be `ClusterNames`? I would expect `Cluster` objects to be returned.

Reply:
That is probably more useful anyway in other cases.